### ISSUE TITLE
Dockerize GCE processing

### ISSUE THREAD

**Issue by zdenulo** [allenday/bigquery-openstreetmap#9]
At the moment, downloading OSM file and converting to geojson is done via startup script (pipeline/gce_download_parse) and bash scripts in osm2bigquery folder. 
It would be convenient to run it as Docker image on GCE. 
Service account would be needed to access GCP stuff from docker container (namely GCS, GCE)

**Comment by allenday** [comment]
@zdenulo is startup script checked in so someone can use it as a starting point for a Dockerfile?

**Comment by zdenulo** [comment]
@allenday yes, startup script is in pipeline/gce_download_parse/startup.sh and there is also process.sh in the same folder
